extensions/utils/RaftLogEntry.js

```diff
@@ -0,0 +1,55 @@
+const { usersBucket } = require('arsenal').constants;
+
+class RaftLogEntry {
+
+    /**
+     * method to format an entry for object metadata
+     *
+     * @param {object} objectMd - object info to format entry
+     * @param {string} bucketPrefix - prefix for bucketname to avoid name clash
+     * @return {object} JSON.stringified entry value to be sent to kafka
```

Comment:
The description is a little bit off (or confusing), what is returned is a regular JS object that contains the stringified JSON kafka message as its `value` attribute.